### PR TITLE
[Rule Tuning] Simple KQL to EQL Conversion

### LINKED ISSUE #3392: [Rule Tuning] Convert simple KQL Windows rules to EQL
## Description

We should evaluate the conversion from KQL simple queries (which don't include new terms and threshold rules) to EQL, aiming to solve case-sensitiveness problems and make them more manageable around exclusions.

### PR DESCRIPTION
**Pull request by w0rk3r**
## Issues

Resolves #3392 

## Summary - What I changed

Converts some KQL queries to EQL (in rules that don't benefit from it like `new_terms`/`threshold`) to simplify rule tunings and solve case sensitiveness problems.

## How To Test

Copy and paste the query in the SDE cluster removing the `event.code` for the AD ones.

### CODE REVIEW

**Comment by protectionsmachine** [comment]
## Rule: Tuning - Guidelines

These guidelines serve as a reminder set of considerations when tuning an existing rule.

### Documentation and Context

- [ ] Detailed description of the suggested changes.
- [ ] Provide example JSON data or screenshots.
- [ ] Provide evidence of reducing benign events mistakenly identified as threats (False Positives).
- [ ] Provide evidence of enhancing detection of true threats that were previously missed (False Negatives).
- [ ] Provide evidence of optimizing resource consumption and execution time of detection rules (Performance).
- [ ] Provide evidence of specific environment factors influencing customized rule tuning (Contextual Tuning).
- [ ] Provide evidence of improvements made by modifying sensitivity by changing alert triggering thresholds (Threshold Adjustments).
- [ ] Provide evidence of refining rules to better detect deviations from typical behavior (Behavioral Tuning).
- [ ] Provide evidence of improvements of adjusting rules based on time-based patterns (Temporal Tuning).
- [ ] Provide reasoning of adjusting priority or severity levels of alerts (Severity Tuning).
- [ ] Provide evidence of improving quality integrity of our data used by detection rules (Data Quality).
- [ ] Ensure the tuning includes necessary updates to the release documentation and versioning.

### Rule Metadata Checks

- [ ] `updated_date` matches the date of tuning PR merged.
- [ ] `min_stack_version` should support the widest stack versions.
- [ ] `name` and `description` should be descriptive and not include typos.
- [ ] `query` should be inclusive, not overly exclusive. Review to ensure the original intent of the rule is maintained.

### Testing and Validation

- [ ] Validate that the tuned rule's performance is satisfactory and does not negatively impact the stack.
- [ ] Ensure that the tuned rule has a low false positive rate.

File: rules/windows/persistence_msds_alloweddelegateto_krbtgt.toml [hunk 1]
```diff
@@ -47,11 +47,11 @@ tags = [
     "Data Source: Active Directory",
 ]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
 
 query = '''
-event.action:modified-user-account and event.code:4738 and
-  winlog.event_data.AllowedToDelegateTo:*krbtgt*
+iam where event.action : "modified-user-account" and event.code : "4738" and
```
Comment (Aegrah) [inline rules/windows/persistence_msds_alloweddelegateto_krbtgt.toml:53]:
```suggestion
iam where event.action == "modified-user-account" and event.code == "4738" and
```

**Review by Aegrah — APPROVED** [review]

File: rules/windows/privilege_escalation_group_policy_iniscript.toml [hunk 2]
```diff
@@ -9,7 +9,7 @@ author = ["Elastic"]
 description = "Detects the modification of Group Policy Objects (GPO) to add a startup/logon script to users or computer objects."
 false_positives = ["Legitimate Administrative Activity"]
 index = ["winlogbeat-*", "logs-system.*", "logs-windows.*"]
-language = "kuery"
+language = "eql"
```
Comment (Mikaayenson) [inline rules/windows/privilege_escalation_group_policy_iniscript.toml:12]: Dont forget to update the date in this rule (ci is failing for this reason).

File: rules/windows/privilege_escalation_group_policy_scheduled_task.toml [hunk 3]
```diff
@@ -96,14 +96,25 @@ tags = [
...
-type = "query"
+type = "eql"
 
 query = '''
-(event.code: "5136" and winlog.event_data.AttributeLDAPDisplayName:("gPCMachineExtensionNames" or "gPCUserExtensionNames") and
-   winlog.event_data.AttributeValue:(*CAB54552-DEEA-4691-817E-ED4A4D1AFC72* and *AADCED64-746C-4633-A97C-D61349046527*))
-or
-(event.code: "5145" and winlog.event_data.ShareName: "\\\\*\\SYSVOL" and winlog.event_data.RelativeTargetName: *ScheduledTasks.xml and
-  (message: WriteData or winlog.event_data.AccessList: *%%4417*))
+any where host.os.type == "windows" and event.code in ("5136", "5145") and
```
[the remark this answers is not in the document]
Comment (w0rk3r) [reply rules/windows/privilege_escalation_group_policy_scheduled_task.toml:102]: It doesn't seem to be that consistent between integration versions

**Review by terrancedejesus — APPROVED** [review]

**Pull request merged**